integration/cosmostests/LICENSE

```diff
@@ -0,0 +1,204 @@
+Cosmos-SDK
+License: Apache2.0
```

Comment:
really think this is necessary?

Reply:
When in doubt, it's typically safer to preserve and not potentially breach copyright licenses. 

I doubt the Cosmos team will take issue, but learning from the k8s community, not preserving Licenses correctly is a legal quagmire.